gen3.biodatacatalyst.nhlbi.nih.gov/manifest.json

```diff
@@ -140,8 +140,7 @@
             "value": "file"
           },
           {
-            "name": "EXTRA_NODES",
-            "value": ""
```

Comment:
I think we actually need this empty `value` property to be here @themarcelor. I don't know what sower's behavior is if the `value` property is empty, and it needs to be set to an empty string `""` to bypass some quirks in pelican-export. (I know). I'm gonna restore it but if this was removed by an auto-linter it might break something next time we try to release

Reply:
@mattreex delivered a change recently that introduces some new behaviors for empty parameters.
I've set him as a reviewer to obtain more comments on that.
If this is truly harmful @mpingram, can you please push a commit to fix that? 🤦 We will need to review the automation if that's the case.